AnnService/inc/Helper/AsyncFileReader.h

```diff
@@ -144,7 +144,7 @@ namespace SPTAG
                 std::uint32_t maxWriteRetries = 2,
                 std::uint16_t threadPoolSize = 4)
             {
-                m_fileHandle.Reset(::CreateFileA(filePath,
+                m_fileHandle.Reset(::CreateFile(filePath,
```

Comment:
If we are going to do this change, maybe we should explicitly test with both unicode and multi-byte configurations? I think currently the test suite only runs multi-byte.

Reply:
We do not build unicode configuration?